src/main/java/plugily/projects/buildbattle/arena/states/guess/InGameState.java

```diff
@@ -333,11 +337,14 @@ private void checkPlayerOutSidePlot(GuessArena pluginArena) {
     if(pluginArena.getArenaOption("IN_PLOT_CHECKER") >= 3) {
       pluginArena.setArenaOption("IN_PLOT_CHECKER", 0);
       for(Player player : pluginArena.getPlayersLeft()) {
-        User user = getPlugin().getUserManager().getUser(player);
         Plot buildPlot = pluginArena.getPlotFromPlayer(player);
+        if (buildPlot == null) {
```

Comment:
can we rename buildPlot to playerPlot here too?

Reply:
Is a null check really required for playersLeft players?

Reply:
> Is a null check really required for playersLeft players?

Yea, `GuessArena#getPlotFromPlayer` method can return null if the `plotList` does not contain the given Player from GuessArena.

Reply:
I mean in general yea its true it can return null but if we only get playersleft its players which are ingame and also playing wiht a plot, normally there shouldn't be a case that the plot is null as if it is the plugin has a issue on another line? I mean yeah it could be there just for reasons but not sure if we oversee another issue if this is skipped and not thrown as error

Reply:
Well, based on my 5 years of experience, in such cases I always prefer to check that the task will definitely never return that value and I have also proven it with tests, but it can happen to someone after some reproduction. I'd rather play it safe and write more code than cause more problems in the future.